.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+name: Test Refactor
+on:
```

Comment:
Could this all be collapsed to `on: [pull_request]`?

Reply:
Yes it could, I keep this to copy the same behavior as it was used in travis. But I'm not sure if I remove push on branch it will work with new commits on pull request, let me try.

Reply:
Yes, it works, I'll let like this. If it's the case the branch part can be included back.